src/chrome-launcher.ts

```diff
@@ -332,7 +332,7 @@ class Launcher {
         log.log('ChromeLauncher', `Killing Chrome instance ${this.chrome.pid}`);
         try {
           if (isWindows) {
-            execSync(`taskkill /pid ${this.chrome.pid} /T /F`);
+            execSync(`taskkill /pid ${this.chrome.pid} /T /F`, {stdio: 'pipe'});
```

Comment:
add a comment that basically says while pipe is the default, stderr also gets printed to process.stderr if you don't explicitly set it? I could see someone forgetting this and removing it in a driveby someday :)

Reply:
Sure, feels safer :)